airbyte-commons-worker/src/main/java/io/airbyte/workers/process/DockerProcessFactory.java

```diff
@@ -41,6 +41,10 @@ public class DockerProcessFactory implements ProcessFactory {
   private static final Path DATA_MOUNT_DESTINATION = Path.of("/data");
   private static final Path LOCAL_MOUNT_DESTINATION = Path.of("/local");
   private static final String IMAGE_EXISTS_SCRIPT = "image_exists.sh";
+  private static final String DD_ENV_VAR =
+      "-XX:+ExitOnOutOfMemoryError -javaagent:/airbyte/dd-java-agent.jar -Ddd.profiling.enabled=true -XX:FlightRecorderOptions=stackdepth=256 -Ddd.trace.sample.rate=30 -Ddd.trace.request_header.tags=User-Agent:http.useragent";
+  private static final List<String> DATADOG_SUPPORT_IMAGES = List.of("source-postgres");
```

Comment:
For future consideration:  we should probably move this list out into an env var to make it easy to update if/when we add more connectors to the ones that we want to enable metrics for.